static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.js

```diff
@@ -102,6 +106,9 @@ jest.mock('commercial/modules/dfp/load-advert', () => ({
 jest.mock('@guardian/consent-management-platform', () => ({
     onConsentChange: jest.fn(),
     getConsentFor: jest.fn(),
+    cmp: {
+        willShowPrivacyMessage: jest.fn(),
```

Comment:
`cmp.willShowPrivacyMessage ` needs to be mocked for `prepareGoogleTag` to resolve.